deploy/kubernetes/base/csidriver.yaml

```diff
@@ -6,3 +6,4 @@ metadata:
   name: fsx.csi.aws.com
 spec:
   attachRequired: false
+  fsGroupPolicy: ReadWriteOnceWithFSType
```

Comment:
Same comment as here: https://github.com/kubernetes-sigs/aws-fsx-csi-driver/pull/272#discussion_r1059150860 . Just want to make sure there's no unintended consequences of setting this.